modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -403,6 +446,80 @@ Users who can access the Windows Network Drive storage via share won't receive a
 
 NOTE: `wnd.activity.sendToSharees` key depends on the `wnd.activity.registerExtension` key to take effect.
 
+=== Collaborative WND
+
+CWND can only be set by an admin in menu:Settings[Admin > Storage]. This mount type cannot be selected by users in the user section. To prepare access for your mount point using the CWND mount type, you must provide a _Service Account_ (SA) which is an oridnary SMB user granting read access to the share you want to mount. You can use one SA for all CWND mounts or separate ones. The SA is used to gather the contents of a share, used by the WND Listener and provides a common `file_id` to all accessing users, while the accessing user has only access to those files and folders he is granted for.
```

Comment:
"which is an ordinary SMB user"

Reply:
How to describe/name a user which is created in AD or in Samba which can access file services but who is not created in ownCloud?

Reply:
I think that's fine. This was just for the typo :)